docs/assets/templates/tools/.dockstore.yml

```diff
@@ -0,0 +1,60 @@
+# The first line refers to the version 1.2 of the .dockstore.yml schema
+version: 1.2
+
+# An array of tools. Each element corresponds to a tool entry on Dockstore.
+tools:
+  
+  # The optional name for a tool, which may only consist of alphanumerics
+  # and internal underscores and hyphens. Names may not exceed 256 characters. 
+  # If using a ``/.dockstore.yml`` with multiple tools, this field is required 
+  # to uniquely identify tools in the repository. Otherwise, it is optional.
+  # 
+  # It should be noted that having the name come first is an arbitrary decision.
+  # You could use subclass instead, for instance. Provided arrays are not broken
+  # up, the order of fields within a .dockstore.yml is not important.
+  - name: <String>
+    
+    # The descriptor language used for the tool. As a reminder, we generally advise against WDL tools.
```

Comment:
`As a reminder, we generally advise against WDL tools.` might be redundant. 
After https://github.com/dockstore/dockstore/commit/7ab7f6d7fa04cfd7011754d4ca0df9c1fce07d83 this shouldn't be possible anyway

Reply:
Will that be merged as part of 1.12?

Reply:
Yep, there's a row of tags that you can expand with ellipsis to see where something is headed/merged
![Screenshot from 2022-03-22 17-35-27](https://user-images.githubusercontent.com/1730679/159580254-dea03e04-3d82-429f-bf32-d4333a9bbc9c.png)

Reply:
I removed the WDL option from the template and changed the table in migrating-tools-to-github-apps.rst to clearly say that WDL tools are not supported via .dockstore.yml registration. Other parts of the docs already note that WDL tools are quasi-deprecated and a previous PR replaced the "getting started with WDL" section to be a workflow instead of a tool, so I think I covered our bases here.